app/models/member.rb

```diff
@@ -121,6 +121,26 @@ def user_has_namespace_maintainer_access?(user, namespace)
                        access_level: Member::AccessLevel::MAINTAINER)
       end
     end
+
+    def membership_source(namespace, user)
```

Comment:
Please pass `member` instead of `user`

Reply:
Updated in 0379505201abd0d8b0a317f46c1819e1c82f5c3e